Smt/Reconstruct/Real/Polynorm.lean

```diff
@@ -5,7 +5,8 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Abdalrhman Mohamed
 -/
 
-import Mathlib.Data.Real.Basic
+import Mathlib.Data.Real.Star
+import Mathlib.Tactic.Ring.RingNF
```

Comment:
Without this, compilation fails with the following error:
```
unknown namespace 'Mathlib.Tactic.RingNF'
```
It looks like `Mathlib.Data.Real.Basic` (which used to include `Mathlib.Data.Real.Star`) transitively imported it, but no longer does so now.